src/data/Philippines-ministry-of-energy_metadata.json

```
@@ -0,0 +1,128 @@
+[
+  {
+    "id": "PMoE-Reference-2023",
+    "publisher": "Philippines - Ministry of Energy",
+    "name": "Reference",
+    "description": "Reference Scenario provide a scenario where current energy policies are maintained in Philippines",
+    "regions": ["Philippines"],
```

```suggestion
    "regions": ["PH"],
```

The Philippines are a country, so I guess we should use an ISO-2 code if we want to be consistent with the IEA merge.